### PR TITLE
fix(test): Don't error on missing op details

### PR DESCRIPTION
**Pull request by AEtheve**
Fixes: #14144

### CODE REVIEW

**Comment by kt3k** [comment]
Can you add test cases?

File: runtime/js/40_testing.js [hunk 1]
```diff
@@ -198,15 +198,16 @@
           }
           ArrayPrototypePush(details, message);
         } else if (dispatchedDiff < completedDiff) {
-          const [name] = OP_DETAILS[key];
+          const [name, hint] = OP_DETAILS[key] || [key, null];
```
Comment (kt3k) [inline runtime/js/40_testing.js:201]: This exposes the name of the op like the below error message:

```
- 1 async operation to op_write was started before this test, but was completed during the test. Async operations should not complete in a test if they were not started in that test.
```

I think the proper fix here is to add the entry for `op_write` in `OP_DETAILS` table.

Comment (bartlomieju) [reply runtime/js/40_testing.js:201]: This is actually fine, I thought I fixed it in https://github.com/denoland/deno/pull/14074 but I missed this

Comment (bartlomieju) [reply runtime/js/40_testing.js:201]: You are right though that we should fill in details for not listed ops.

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**